willyserge/broadcaster
# #169853168 A user should be able to create a red-flag record

#### What does this PR do?
have a working endpoint for getting red-flag records created by a user
#### Description of Task to be completed?
 working endpoint   `/api/v1/red-flags`
#### How should this be manually tested?
- run npm install to install all dependencies
- run npm start and server
- see your port number the server is running on
- go to postman and make a `POST request` to` http://localhost: port number /api/v1/red-flags`

#### What are the relevant pivotal tracker stories?
`#169853168`
